app/helpers/gradebook_helper.rb

```diff
@@ -71,11 +71,12 @@ def gradebook_rows(matrix, course, section = nil, lecture = nil)
       grace_days = 0
       late_days = 0
 
+      # CGI.escapeHTML to avoid XSS
```

Comment:
Note: We escape here because in `gradebooks/view.html.erb`, we use `<%== gradebook_rows(...).to_json %>`. That is, we output the raw html (`<%== ... %>`) and we don't have the opportunity to escape the string then.